### PR TITLE
feat: hide breaking news

### PR DESCRIPTION
**Pull request by 0xrxL**

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/patches/HideBrekingNewsShelfPatch.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package app.revanced.integrations.patches;
+
+import android.view.View;
+
+import app.revanced.integrations.adremover.AdRemoverAPI;
+import app.revanced.integrations.settings.SettingsEnum;
+
+public class HideBrekingNewsShelfPatch {
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/HideBrekingNewsShelfPatch.java:8]: Misspelled "Breaking" here.

File: app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java [hunk 2]
```diff
@@ -83,6 +83,7 @@ public enum SettingsEnum {
     HIDE_SHORTS_COMMENTS_BUTTON("revanced_hide_shorts_comments_button", false, ReturnType.BOOLEAN),
     HIDE_TIME_AND_SEEKBAR("revanced_hide_time_and_seekbar", false, ReturnType.BOOLEAN),
     HIDE_WATCH_IN_VR("revanced_hide_watch_in_vr", false, ReturnType.BOOLEAN, true),
+    HIDE_BREAKING_NEWS_SHELF("revanced_hide_hide_breaking_news_shelf", false, ReturnType.BOOLEAN, true),
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java:86]: Duplicate settings name "hide_hide".

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java [hunk 3]
```diff
@@ -83,6 +83,7 @@ public enum SettingsEnum {
     HIDE_SHORTS_COMMENTS_BUTTON("revanced_hide_shorts_comments_button", false, ReturnType.BOOLEAN),
     HIDE_TIME_AND_SEEKBAR("revanced_hide_time_and_seekbar", false, ReturnType.BOOLEAN),
     HIDE_WATCH_IN_VR("revanced_hide_watch_in_vr", false, ReturnType.BOOLEAN, true),
+    HIDE_BREAKING_NEWS_SHELF("revanced_hide_breaking_news_shelf", false, ReturnType.BOOLEAN, true),
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java:86]:
```suggestion
    HIDE_BREAKING_NEWS_SHELF("revanced_hide_breaking_news_shelf", true, ReturnType.BOOLEAN, true),
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Comment by 0xrxL** [comment]
Ok, now it's all ready. I mistakenly omitted some changes.

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**